src/main/java/wolox/training/repositories/IUserRepository.java

```diff
@@ -0,0 +1,8 @@
+package wolox.training.repositories;
+
+import org.springframework.data.repository.CrudRepository;
+import wolox.training.models.Users;
+
+public interface IUserRepository extends CrudRepository<Users, Long> {
```

Comment:
Here is missing the repository annotation
```suggestion
@Repository
public interface IUserRepository extends CrudRepository<Users, Long> {
```

Reply:
Additionally, the method to get a user by username is missing.